src/components/ItaliaTheme/Blocks/Listing/Commons/ListingImage.jsx

```diff
@@ -30,8 +30,12 @@ const ListingImage = ({
   // photogallery needs to check for null image
   // https://stackoverflow.com/questions/33136399/is-there-a-way-to-tell-if-reactelement-renders-null
 
-  const image = (
-    <UniversalLink item={item}>{Image(commonImageProps)}</UniversalLink>
+  const image = !imageProps.noWrapLink ? (
```

Comment:
Bene che hai aggiunto un parametro per non mettere il link. Io però lo estrarrei dalle props invece di lasciarlo in `imageProps` per non inquinare i parametri passati proprio all'immagine. In altri termini, lo destrutturerei alla riga 13 prima di `...imageProps`

Reply:
Fatto!